### ISSUE TITLE
Profile selection should default to "Default"

### ISSUE THREAD

**Issue by tflo** [t-mart/ItemVersion#37]
### What version of ItemVersion are you running?

2022.47.2

### What is your issue?

While it is always good to have the option to set a char/class-specific profile, it would be nice if the profile selection would default to "Default" instead of "\<char name\>".

This would save us multi-alt players from having to adjust the profile selection 20 times ;) 

I don't think that someone really prefers to have different tooltip modifier keys for their toons, or a short version tooltip for one class and a detailed version tooltip for another… And if someone wants, he still can set a specific profile.

(It wouldn't "complain" if the addon was about char/class/spec-specific things like eg spells, auras, action bars etc. But this is not the case here.)

Thanks for considering,

– Tom

**Comment by t-mart** [comment]
This entire options dialog comes from Ace.

I'm not sure if there is a way to configure for the default profile. But, I'll look into it.

Do you know of any other addons that do this, for reference?

**Comment by tflo** [comment]
> I'm not sure if there is a way to configure for the default profile. 

You just have to activate it in your Init.lua, line 10:

```diff
10c10
<   self.db = LibStub("AceDB-3.0"):New("ItemVersionDB", self:GetDefaultDB())
---
>   self.db = LibStub("AceDB-3.0"):New("ItemVersionDB", self:GetDefaultDB(), true)
```

See also https://www.wowace.com/projects/ace3/pages/api/ace-db-3-0

**Comment by t-mart** [comment]
Implemented in #42.

I agree with your reasoning on this: this addon doesn't really seem like one where per-character configuration will be common.

(Again, if you want to put your name on the AUTHORS list in the root directory, make a PR.)

**Comment by tflo** [comment]
Actually I think there are very few addons where per-char/spec/class profiles make sense. [Raven](https://www.curseforge.com/wow/addons/raven) would be a good example (though it also works fine with a shared profile, since it has other means to make icons visible only for specific classes), or [Clicked](https://www.curseforge.com/wow/addons/clicked), depending for what you use it, and similar things.

[TomTom](https://www.curseforge.com/wow/addons/tomtom) comes with two profiles, one for the settings, and one for the waypoints, which is good because for the waypoints it can be useful to have them per-char, whereas for the options it would be nonsense to use per-char profiles.

Thanks for implementing it! Also for the Command key addition from the other thread of course.

– Tom
